cypress/integration/common/metadata.js

```diff
@@ -3,7 +3,59 @@
  * This file contains all the steps for asserting metadata.
  *****************/
 import { Given, When, Then } from "cypress-cucumber-preprocessor/steps";
+import { Beacon } from "../../utils";
 
 Then("browser title is {string}", (title) => {
   cy.title().should('eq', title);
+});
+
+Given('user is navigating to {string}', (a) => {
+  cy.visit(a);
+});
+
+Then('the common load event properties are captured',()  => {
+  
+cy.window().then((win) => {
+   //create a map of request being sent of
+   var networkrequests = win.performance.getEntries().map(r => r.name);
+   //list of URLs collected
+   var urlList = new Array();
+   for(var i=0; i<networkrequests.length; i++){
+      var temp = networkrequests[i].toString()
+    if(temp.includes('b/ss')){
+       //find the GA url and store it
+        urlList.push(networkrequests[i]);
+     }
+   }   
+ let analyticObject = new Beacon(urlList[urlList.length-1]);
```

Comment:
Remember const unless you are going to modify it.

Reply:
Const it is!